pages/pil/read/index.js

```diff
@@ -8,7 +8,7 @@ module.exports = settings => {
   });
 
   app.get('/', (req, res, next) => {
-    if (req.pil.status !== 'active') {
+    if (req.pil.status === 'pending') {
```

Comment:
is `pending` the status we give a PIL when it's created but not submitted? I thought we usually defaulted to inactive